File: Phasegate/DayOne/Level 2/java/TEST/TestTaskOne.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
public class TestTaskOne{
    @Test
    public void testThatZerosAreMovedToEnd() {
        int[] number   = { 5, 0, 3, 0, 2, 0 };
        int[] actual   = TaskFive.moveZerosToEnd(number);
        int[] expected = { 5, 3, 2, 0, 0, 0 };
        assertArrayEquals(expected, actual);
    }
}

File: Phasegate/DayOne/Level 2/python/TEST/test_task_one.py
from unittest import TestCase
import task_one
class task_one_test(TestCase):
    def test_that_zeros_are_moved_to_end(self):
        numbers  = [5, 0, 3, 0, 2, 0]
        actual   = task_one.move_zeros_to_end(numbers)
        expected = [5, 3, 2, 0, 0, 0]
        self.assertEqual(actual, expected)
